examples/Example4/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: 2020 CERN
+# SPDX-License-Identifier: Apache-2.0
+
+# Example 2 of particle processing with GPU, including geometry
+# and reproducible results using one RANLUX++ state per track.
```

Comment:
Example 4 of particle processing with GPU, including geometry and magnetic field propagation.

Reply:
Thanks.